.github/labeler.yml

```diff
@@ -0,0 +1,20 @@
+version: 1
+appendOnly: true
+# Labels are applied based on conventional commits standard
+# https://www.conventionalcommits.org/en/v1.0.0/
+# These labels are later used in release notes. See .github/release.yml
+labels:
+# If the PR title has an ! before the : it will be considered a breaking change
+# For example, `feat!: add new feature` will be considered a breaking change
+- label: breaking-change
+  title: "^[^:]+!:.*"
+- label: enhancement
+  title: "^feat(\\(.+\\))?!?:.*"
+- label: bug
+  title: "^fix(\\(.+\\))?!?:.*"
+- label: documentation
+  title: "^docs(\\(.+\\))?!?:.*"
+- label: performance
+  title: "^perf(\\(.+\\))?!?:.*"
+- label: ci
```

Comment:
I tend to use `chore` for "not something the user cares about".  Should we add `chore` to this list or should I start using `ci`?

Note: from the conventional commits page there is also `build`, `style`, and `test` (users may care about style but not the other two) but I don't tend to use those myself (I'd probably lump build and test under `chore`) so I'm good either way.

Reply:
Okay I've set those other ones to map to a `chore` label, and set the release notes to ignore that.